diff --git a/node_modules/vue-tabs-component/dist/index.js b/node_modules/vue-tabs-component/dist/index.js
index d285ef4..8216328 100644
--- a/node_modules/vue-tabs-component/dist/index.js
+++ b/node_modules/vue-tabs-component/dist/index.js
@@ -361,10 +361,10 @@ var __WEBPACK_AMD_DEFINE_FACTORY__, __WEBPACK_AMD_DEFINE_ARRAY__, __WEBPACK_AMD_
             },
             hash: function hash() {
                 if (this.isDisabled) {
-                    return '#';
+                    return '';
                 }
 
-                return '#' + this.computedId;
+                return '' + this.computedId;
             }
         }
     };
@@ -499,8 +499,8 @@ var __WEBPACK_AMD_DEFINE_FACTORY__, __WEBPACK_AMD_DEFINE_ARRAY__, __WEBPACK_AMD_
                 return;
             }
 
-            if (this.options.defaultTabHash !== null && this.findTab("#" + this.options.defaultTabHash)) {
-                this.selectTab("#" + this.options.defaultTabHash);
+            if (this.options.defaultTabHash !== null && this.findTab("" + this.options.defaultTabHash)) {
+                this.selectTab("" + this.options.defaultTabHash);
                 return;
             }
 
diff --git a/node_modules/vue-tabs-component/src/components/Tab.vue b/node_modules/vue-tabs-component/src/components/Tab.vue
index 31d7cc1..103c9f0 100644
--- a/node_modules/vue-tabs-component/src/components/Tab.vue
+++ b/node_modules/vue-tabs-component/src/components/Tab.vue
@@ -35,10 +35,10 @@
 
             hash() {
                 if (this.isDisabled) {
-                    return '#';
+                    return '';
                 }
 
-                return '#' + this.computedId;
+                return '' + this.computedId;
             },
         },
     };
diff --git a/node_modules/vue-tabs-component/src/components/Tabs.vue b/node_modules/vue-tabs-component/src/components/Tabs.vue
index 2a87c6c..7d18a02 100644
--- a/node_modules/vue-tabs-component/src/components/Tabs.vue
+++ b/node_modules/vue-tabs-component/src/components/Tabs.vue
@@ -75,8 +75,8 @@
                 return;
             }
 
-            if(this.options.defaultTabHash !== null && this.findTab("#" + this.options.defaultTabHash)) {
-                this.selectTab("#" + this.options.defaultTabHash);
+            if(this.options.defaultTabHash !== null && this.findTab("" + this.options.defaultTabHash)) {
+                this.selectTab("" + this.options.defaultTabHash);
                 return;
             }
 
